i18n-helpers/src/lib.rs

```diff
@@ -126,19 +126,32 @@ pub fn extract_events<'a>(text: &'a str, state: Option<State<'static>>) -> Vec<(
 }
 
 /// Markdown events grouped by type.
-#[derive(Debug, Copy, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
 pub enum Group<'a> {
     /// Markdown events which should be translated.
     ///
     /// This includes `[Text("foo")]` as well as sequences with text
     /// such as `[Start(Emphasis), Text("foo") End(Emphasis)]`.
-    Translate(&'a [(usize, Event<'a>)]),
+    Translate(&'a [(usize, Event<'a>)], GroupExtra),
 
     /// Markdown events which should be skipped when translating.
     ///
     /// This includes structural events such as `Start(Heading(H1,
     /// None, vec![]))`.
-    Skip(&'a [(usize, Event<'a>)]),
+    Skip(&'a [(usize, Event<'a>)], GroupExtra),
+}
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct GroupExtra {
+    translator_comments: Vec<String>,
+}
+
+impl Default for GroupExtra {
+    fn default() -> GroupExtra {
+        GroupExtra {
+            translator_comments: vec![],
+        }
+    }
```

Comment:
This looks like the same as what `#[derive(Default)]` on `GroupExtra` would give you.

Reply:
Acknowledged: switched to derived impl.